### PR TITLE
fix: [CDS-34185]: createCall for creation of services/Env instead of updateCall

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
##### Summary:

Earlier we were using updateCall for the creation of service/env also.
now switched to respective useCreateServiceV2/useCreateEnvironmentV2 calls.

##### Jira Links:

https://harness.atlassian.net/browse/CDS-34185

##### Screenshots:

After changes - 

https://user-images.githubusercontent.com/98325173/155966212-b54053b3-f02c-4b61-a087-21843b7fee80.mov

https://user-images.githubusercontent.com/98325173/155966231-7b0d8f67-26e8-40d5-8066-5c2375c24ab8.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

**Comment by Sagar-Rajput7** [comment]
retrigger typecheck

**Comment by Sagar-Rajput7** [comment]
retrigger coverage

**Comment by Sagar-Rajput7** [comment]
trigger rebase

**Comment by Sagar-Rajput7** [comment]
retrigger jest

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Review by shamala-harness — APPROVED** [review]

File: src/modules/75-cd/components/Environments/EnvironmentList/EnvironmentsList.tsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const EnvironmentList: React.FC = () => {
                 : { name: '', identifier: '', orgIdentifier, projectIdentifier }
             }
             isEdit={editable}
-            isEnvironment
+            isEnvironment={!editable}
```
Comment (samarth-sharma31) [inline src/modules/75-cd/components/Environments/EnvironmentList/EnvironmentsList.tsx:95]: Hey, lets connect regarding this - just wanted to understand the use of "isEnvironment" boolean in file --> src/modules/75-cd/components/Environments/EnvironmentList/EnvironmentsModal.tsx

As isEdit kindof also contains the similar info and EnvironmentsModal is only being used in EnvironmentsList.tsx

**Comment by Sagar-Rajput7** [comment]
retrigger jest

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Comment by Sagar-Rajput7** [comment]
trigger rebase

**Comment by Sagar-Rajput7** [comment]
trigger rebase

**Comment by Sagar-Rajput7** [comment]
trigger cypress

File: src/modules/75-cd/components/Services/ServicesListPage/ServiceModal.tsx [hunk 2]
```diff
@@ -102,11 +107,7 @@ export const NewEditServiceModalYaml: React.FC<NewEditServiceModalPropsYaml> = (
             onCreateOrUpdate(values)
           }
         } else {
-          const response = await updateService({
-            ...omit(values, 'accountId', 'deleted'),
-            orgIdentifier,
-            projectIdentifier
-          })
+          const response = await createService([{ ...values, orgIdentifier, projectIdentifier }])
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/Services/ServicesListPage/ServiceModal.tsx:110]: Earlier **accountId** and **deleted** were omitted, now they are not omitted. Hope that is intended change.

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/75-cd/components/Services/ServicesListPage/ServiceModal.tsx (hunk 2 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/Services/ServicesListPage/ServiceModal.tsx:110]: Yeah, for creation delete is defaulted to false and accountID is auto picked for the creationCall (please see the attached image of request call while creation)

image

Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/Services/ServicesListPage/ServiceModal.tsx:110]: Thanks

**Review by samarth-sharma31 — APPROVED** [review]

**Pull request merged**